### PR TITLE
Add observability-policies app

### PR DESCRIPTION
**Pull request by marieroque**
towards https://github.com/giantswarm/giantswarm/issues/31155

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.

### Testing

Description on how default-apps-vsphere can be tested.

- [ ] fresh install works
- [ ] upgrade from previous version works

### Trigger e2e tests

/run cluster-test-suites

### CODE REVIEW

File: helm/default-apps-vsphere/values.yaml [hunk 1]
```diff
@@ -181,6 +181,18 @@ apps:
...
+    appName: observability-policies
+    chartName: observability-policies
+    catalog: default
+    clusterValues:
+      configMap: false
+      secret: false
+    dependsOn: kyverno-crds
+    namespace: kube-system
+    # used by renovate
+    # repo: giantswarm/observability-policies
```
Comment (gawertm) [inline helm/default-apps-vsphere/values.yaml:194]:
```suggestion
    # repo: giantswarm/observability-policies-app
```

Comment (gawertm) [reply helm/default-apps-vsphere/values.yaml:194]: not sure this really matters, but seems like the repo is called "observability-policies-app"?

**Review by glitchcrab — APPROVED** [review]

**Comment by glitchcrab** [comment]
i've added the skip/ci flag as the tests have already passed

**Pull request merged**